support/modules/rootfs-image

```diff
@@ -58,6 +58,11 @@ parse_conf_file() {
         return 1
     fi
 
+    # For UBI, standardize on the `/dev/` variant. The kernel only accepts an argument without
+    # `/dev/`, but all userspace tools use the `/dev/` variant.
+    MENDER_ROOTFS_PART_A="${MENDER_ROOTFS_PART_A/#ubi/\/dev\/ubi}"
```

Comment:
Q: Does this even work with regular old `sh`?

Reply:
I thought this was a bashism

Reply:
Darn, you're right, it is. Fixing.